doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -37,10 +37,10 @@ following Features tabs defined here.
 | Conformance                            | X             | Kubernetes conformance test |
 | None                                   | X             | Kubernetes mainstream features |
 | Feature:ComprehensiveNamespaceDraining | X             | Namespaces should always delete fast (ALL of 100 namespaces in 150 seconds) |
```

Comment:
What is the link to this? Is there a namespace drain or just drain node?

Reply:
I could not find any link to this. I think this is not a feature gate.

Reply:
How about this one: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#rollingupdatedaemonset-v1-apps

Reply:
@collivier Can you help? Or should should this be "node draining"